packages/starlight/translations/he.json

```diff
@@ -0,0 +1,22 @@
+  {
+    "skipLink.label": "דלגו לתוכן",
+    "search.label": "חיפוש",
+    "search.shortcutLabel": "(לחצו / לחיפוש)",
+    "search.cancelLabel": "ביטול",
+    "search.devWarning": "לא ניתן להציג תצוגה מקדימה של האתר באופן מקומי.",
+    "themeSelect.accessibleLabel": "בחרו פרופיל צבע",
+    "themeSelect.dark": "כהה",
+    "themeSelect.light": "בהיר",
+    "themeSelect.auto": "מערכת",
+    "languageSelect.accessibleLabel": "בחרו שפה",
+    "menuButton.accessibleLabel": "תפריט",
+    "sidebarNav.accessibleLabel": "ראשי",
+    "tableOfContents.onThisPage": "בדף זה",
+    "tableOfContents.overview": "סקירה כללית",
+    "i18n.untranslatedContent": "תוכן זה אינו זמין עדיין בשפת העברית.",
```

Comment:
Might be Google Translate tripping me up, but testing there, this translates as “This content is not yet available **in Hebrew**.” but in English we use “This content is not available in **your language** yet.”

This is a subtle difference but it’s hypothetically possible depending on site set up that this notice might be displayed in Hebrew to someone who wants content in a different language, so should be translated more generically as “your language”.

Reply:
done